lib/tasks/webpacker/yarn_install.rake

```diff
@@ -1,6 +1,6 @@
 namespace :webpacker do
   desc "Support for older Rails versions. Install all JavaScript dependencies as specified via Yarn"
   task :yarn_install do
-    system "yarn install --no-progress --frozen-lockfile --production"
+    system "yarn install --no-progress --frozen-lockfile --production --network-timeout 1000000"
```

Comment:
How about making the rake task accept arguments and appending them to the `system` call. 

Would be more flexible IMO.

Reply:
This fails during deploy using a buildpack. I am not sure if it is possible to specify a concrete rake task arguments when using a buildpack.